modules/entity_meta_relation_node/config/schema/entity_meta.entity_type.schema.yml

```diff
@@ -0,0 +1,12 @@
+entity_meta.entity_meta_type.*:
```

Comment:
This schema definition belongs in the main module not the node module. That is where the config entity is defined.